js/sr_tools_db.js

```diff
@@ -1722,91 +1731,629 @@ var db = {
+				res.attributes.logic += -1;
+				res.max_attributes.logic += -1;
+				res.attributes.charisma += -1;
+				res.max_attributes.charisma += -1;
+				res.attributes.magic += 6;
+				break;
+
+			case 'Fomoraig':
+				res.attributes.body += 4;
+				res.min_attributes.body += 4;
+				res.max_attributes.body += 4;
+				res.attributes.agility += -1;
+				res.max_attributes.agility += -1;
+				res.attributes.strength += 4;
+				res.min_attributes.strength += 4;
+				res.max_attributes.strength += 4;
+				res.attributes.logic += -1;
+				res.max_attributes.logic += -1;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -2;
+				res.max_attributes.charisma += -2;
+				res.attributes.magic += 6;
+				break;
+
+			case 'Mutaqua':
+				res.attributes.body += 4;
+				res.min_attributes.body += 4;
+				res.max_attributes.body += 4;
+				res.attributes.agility += -1;
+				res.max_attributes.agility += -1;
+				res.attributes.strength += 4;
+				res.min_attributes.strength += 4;
+				res.max_attributes.strength += 4;
+				res.attributes.logic += -1;
+				res.max_attributes.logic += -1;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -2;
+				res.max_attributes.charisma += -2;
+				res.attributes.magic += 6;
+				break;
+
+			case 'Bandersnatch':
+				res.attributes.body += 5;
+				res.min_attributes.body += 5;
+				res.max_attributes.body += 5;
+				res.attributes.strength += 4;
+				res.min_attributes.strength += 4;
+				res.max_attributes.strength += 4;
+				res.attributes.magic += 6;
+				break;
+			case 'Dzoo-Noo-Qua':
+				res.attributes.body += 4;
+				res.min_attributes.body += 4;
+				res.max_attributes.body += 4;
+				res.attributes.agility += -1;
+				res.max_attributes.agility += -1;
+				res.attributes.strength += 4;
+				res.min_attributes.strength += 4;
+				res.max_attributes.strength += 4;
+				res.attributes.logic += -1;
+				res.max_attributes.logic += -1;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -2;
+				res.max_attributes.charisma += -2;
+				res.attributes.magic += 6;
+				break;
+
+			case 'Centaur':
+				res.attributes.body += 2;
+				res.min_attributes.body += 2;
+				res.max_attributes.body += 2;
+				res.attributes.strength += 2;
+				res.min_attributes.strength += 2;
+				res.max_attributes.strength += 2;
+				res.attributes.intuition += -1;
+				res.max_attributes.intuition += -1;
+				res.attributes.charisma += -1;
+				res.max_attributes.charisma += -1;
+				res.attributes.edge += -1;
+				res.max_attributes.edge += -1;
+				res.attributes.magic += 1;
+				res.augmentations.push({name: 'Low-Light Vision'});
+				res.augmentations.push({name: 'Thermographic Vision'});
+				res.augmentations.push({name: 'Magic Sense'});
+				res.augmentations.push({name: 'Natural Weapon (Kick: DV(STR+2)P, AP +1, Reach +1)'});
```

Comment:
If you want the centaur's kick to appear in the list of weapons, you will want to add it to the `_weapon_list` array. Though adding it will make it a selectable weapon for all races, since there is no way to limit weapons to specific races at this time.